force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +299,14 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
+        Integer sum= 0;
+        for (Integer element : numbers) {
+            if (element >0) {
+                sum = sum + element;
+            }
+        }
 
-        // Loop through the list of integers
-            // if the number is negative skip this iteration
-        
-
-        return null; // Replace null with the variable you used to store the result
+        return sum; 
```

Comment:
good work. Again, you can use the more conventional short-hand sum += element. But this is totally fine